src/devices/Qmc5883l/OutputRate.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// Output data rate is controlled by ODR registers. Four data update frequencies can be selected: 10Hz, 50Hz,
+    /// 100Hz and 200Hz.For most of compassing applications, we recommend 10 Hz for low power consumption.For
+    /// gaming, the high update rate such as 100Hz or 200Hz can be used.
```

Comment:
- you should avoid register specific information
- extra info about recommendations should be in remarks sections
- summary should just contain generic info like `Output data rate of the device` or `Number of samples device can produce per second`
- use spaces between dot ending the sentence and first word